ballista/scheduler/src/scheduler_server/grpc.rs

```diff
@@ -603,6 +603,55 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan> SchedulerGrpc
 
         Ok(Response::new(SchedulerLostResponse {}))
     }
+
+    async fn list_jobs(
+        &self,
+        _request: Request<ListJobsRequest>,
+    ) -> Result<Response<ListJobsResponse>, Status> {
+        let now = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .expect("Time went backwards");
+
+        match self.state.task_manager.get_jobs().await {
+            Ok(listed_jobs) => {
+                let mut jobs = Vec::with_capacity(listed_jobs.len());
+
+                for job_overview in listed_jobs.iter().filter(|o| o.is_running()) {
+                    let durations_ms = now
+                        .checked_sub(Duration::from_millis(job_overview.queued_at))
+                        .ok_or_else(|| {
+                            Status::internal(format!(
+                                "invalid queue_at: {}",
+                                job_overview.queued_at
+                            ))
+                        })?
+                        .as_millis() as u64;
+                    let total_task_duration_ms =
```

Comment:
For now this is fine but we'll need to fix this. What we have here will show the wall clock time since the job was started, but what we actually want is the total time of all tasks.

Reply:
I will update it